cparser/Elab.ml

```diff
@@ -2109,7 +2170,6 @@ let elab_fundef env spec name defs body loc =
     | ty, None ->
         (ty, [],env1)
     | TFun(ty_ret, None, false, attr), Some params ->
-        warning loc "Non-prototype, pre-standard function definition.@  Converting to prototype form";
```

Comment:
I'd like to keep this warning in a specific class of warnings, say "CompCert C conformance", which would be off by default.  Other warnings could be put or added to this class to draw attention to ISO C features that are not in the core CompCert C language.